packages/mobile_app/lib/screens/daily/constants.dart

```diff
@@ -27,6 +23,8 @@ const CALENDAR_SEPARATOR_RADIUS = 16.0;
 //
 const SPACE_BETWEEN_CALENDAR_POLLS = 1.0;
 const SPACE_BETWEEN_POLLS_PROPHECY = 8.0;
+const SPACE_AFTER_AMBIANCE = 32.0;
```

Comment:
Why to move this to a constant? Its not common for the app to extract layout params to constants.

Reply:
For a faster corrections with @bbks by the time of release. I can remove it, if you wish

Reply:
Thats fine